rom/dev/src/main.rs

```diff
@@ -184,9 +185,22 @@ pub extern "C" fn rom_entry() -> ! {
 }
 
 fn run_fips_tests(env: &mut KatsEnv) -> CaliptraResult<()> {
+    cprintln!("[kat] ++");
```

Comment:
This is repeating that what's done at the top of kat::execute_kat(). Do we care?